tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -447,6 +458,21 @@ func storeLimitCommandFunc(cmd *cobra.Command, args []string) {
 	}
 }
 
+func storeCheckCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		cmd.Usage()
+		return
+	}
+
+	prefix := storesPrefix + "?state=" + args[0]
```

Comment:
The state can not directly use "Tombstone"/"up"/"offline". See https://github.com/tikv/pd/blob/69209e0a3ec20db90242a9375cf885724440c68e/server/api/store.go#L655. ^ ^
I think we need to check the output result in your tests.

Reply:
Right, thx for your reminder. There is a mapping from string value to numeric value.